broker/api.py

```diff
@@ -115,7 +115,7 @@ def last_operation(
         :rtype: LastOperation
         """
 
-        instance = ServiceInstance.query.get(instance_id)
+        instance = db.session.get(ServiceInstance, instance_id)
```

Comment:
this is a sqlalchemy 1.x to 2.x change - you'll see this pattern a ton. 
`ModelType.query.get(identifier)` is now `db.session.get(ModelType, identifier)`